junit-platform-commons/src/main/java/org/junit/platform/commons/util/AnnotationUtils.java

```diff
@@ -258,6 +258,10 @@ else if (candidate.annotationType().equals(containerType)) {
 		// @formatter:on
 	}
 
+	public static List<Method> findAnnotatedMethods(Class<?> clazz, Class<? extends Annotation> annotationType) {
+		return findAnnotatedMethods(clazz, annotationType, MethodSortOrder.HierarchyDown);
+	}
+
```

Comment:
I removed the new one from `AnnotationUtils` and switched to the one declared in `ReflectionUtils`.